### PR TITLE
chore(components/google-cloud): Update tox test env to Python 3.9

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Updating Nox test environment for GCPC to python 3.9

### CODE REVIEW

File: components/google-cloud/tox.ini [hunk 1]
```diff
@@ -30,7 +32,7 @@ depends =
     report: py38
```
Comment (IronPan) [inline components/google-cloud/tox.ini:32]: do you need to change here

Comment (SinaChavoshi) [reply components/google-cloud/tox.ini:32]: ohh good catch and yes.

**Pull request merged**